OrbitQt/Expected.h

```diff
@@ -0,0 +1,61 @@
+#ifndef ORBITQT_EXPECTED_H_
```

Comment:
This should probably go to OrbitBase.

Needs a test

Reply:
Are you in favour of using this Expected class over https://github.com/ned14/outcome ? I would write tests or use the external library, since it seems well tested.

Reply:
I'd go for StatusOr, used in many Google projects.  @dimitry- , we could steal the one from protobufs maybe?

Reply:
absl also has it now: https://github.com/abseil/abseil-cpp/blob/master/absl/status/status.h

Reply:
actually I stand corrected - there is no statusor, only status

Reply:
We should probably replace it by outcome (https://github.com/ned14/outcome/tree/develop). Single-Header-Lib, Apache-2.0, unit tested, conan- and cmake-support.

Reply:
I used the Outcome class suggested by Henning now